Group-Video/OpenVideoCall-Windows/README.md

```diff
@@ -1,6 +1,6 @@
 # Open Video Call Windows
 
-*其他语言版本： [简体中文](README.zh.md)*
+*Read this in other languages: [Chinese](README.zh.md)*
```

Comment:
missed this one

Reply:
@plutoless I have updated the branch with this fix now.